t/plugin/proxy-mirror.t

```diff
@@ -838,3 +838,66 @@ GET /hello?a=1
 hello world
 --- error_log
 uri: /a/hello?a=1
+
+
+
+=== TEST 30: use proxy-rewrite to change uri before mirror
```

Comment:
Let's move it to a new file.

> When the test file is too large, for example > 800 lines, you should split it to a new file. Please take a look at t/plugin/limit-conn.t and t/plugin/limit-conn2.t.

https://github.com/apache/apisix/blob/master/CONTRIBUTING.md#check-code-style-and-test-case-style

Reply:
moved